Cargo.toml

```diff
@@ -47,6 +47,7 @@ syslog = "6.0.0"
 defaults = "0.2.0"
 freedesktop-icon-lookup = "0.1.0"
 sublime_fuzzy = "0.7.0"
+keyframe = "1.1.1"
```

Comment:
Looks unused

Reply:
Removed

Reply:
And cargo.lock :) apparently need to check that in ci

Reply:
Oops. Sorry about that